app/src/main/java/com/kristallik/jokeapp/data/JokeGenerator.kt

```diff
@@ -1,10 +1,10 @@
 package com.kristallik.jokeapp.data
 
-import kotlin.random.Random
 
-// object, чтобы не создавался при каждом выхове новый экземпляр
+// object, чтобы не создавался при каждом вызове новый экземпляр
 object JokeGenerator {
-    val jokes = ArrayList<Joke>()
+    var isLocalLoaded: Boolean = false
```

Comment:
у котлина есть крутая штука, как геттеры и сеттеры (https://kotlinlang.org/docs/properties.html#getters-and-setters)
и тебе достаточно написать конструкцию типа
```suggestion
val isLocalLoaded: Boolean
        get() = jokes.size != 0
```